project_tier/course_materials/templates/course_materials/course_materials_index_page.html

```diff
@@ -0,0 +1,137 @@
+                </label>
+              </div>
+            {% endfor %}
+          </div>
+        {% endif %}
+
+        {% if course_level_tags %}
+          <div class="tag-group">
+            <h6>Course Level</h6>
+            {% for tag in course_level_tags %}
+              <div class="tag">
+                <label>
+                  <input class="tag-checkbox" data-tag="course-levels" name="{{ tag }}" type="checkbox"{% if tag.name in course_level_tags_checked %} checked{% endif %}>
+                  <span class="tag-name">{{ tag }}</span>
+                </label>
+              </div>
+            {% endfor %}
+          </div>
+        {% endif %}
+
+        {% if protocol_tags %}
+          <div class="tag-group">
+            <h6>Protocol</h6>
+            {% for tag in protocol_tags %}
+              <div class="tag">
+                <label>
+                  <input class="tag-checkbox" data-tag="protocols" name="{{ tag }}" type="checkbox"{% if tag.name in protocol_tags_checked %} checked{% endif %}>
+                  <span class="tag-name">{{ tag }}</span>
+                </label>
+              </div>
+            {% endfor %}
+          </div>
+        {% endif %}
+
+        {% if results.filtered %}
+          <a class="clear-button" href="{{ page.url }}">
+            <i class="fa fa-times-circle"></i> Clear filters
+          </a>
+        {% endif %}
+      </nav>
+    </div>
+  </div>
+{% endblock %}
+
+{% block main_content %}
+  <div class="course-materials-list row">
+    <div class="body-content">
+      {% if results.filtered %}
+        <p class="filter-text">{{ results.hidden }} course materials{{ results.hidden|pluralize:" is,s are" }} hidden. <a href="{{ page.url }}">Clear filters <i class="fa fa-times"></i></a></p>
+      {% else %}
+        <p class="filter-text">{{ results.shown }} course{{ results.shown|pluralize}} found.</p>
+      {% endif %}
+      <div class="course-materials">
+        {% for course in course_materials %}
+          <h2>{{ course.specific.course_title }}, {{course.specific.university }}</h2>
+          <div class="tags">
+            {% for tag in course.specific.discipline_tags.all %}
+              <a class="tag" href="?disciplines={{ tag }}">{{ tag }}</a>
+            {% endfor %}
+            {% for tag in course.specific.course_level_tags.all %}
+              <a class="tag" href="?course-levels={{ tag }}">{{ tag }}</a>
+            {% endfor %}
+            {% for tag in course.specific.protocol_tags.all %}
+              <a class="tag" href="?protocols={{ tag }}">{{ tag }}</a>
+            {% endfor %}
+          </div>
+          <h3>{{course.specific.course_name}}, {{course.specific.semester }}</h3>
+          <h3>Instructor: {{ course.specific.instructor }}</h3>
+          <p class="excerpt">{{ course.specific.course_description }}</p>
+        {% endfor %}
+      </div>
+    </div>
+  </div>
+{% endblock %}
+
+{% block extra_js %}
+  <script>
```

Comment:
Did you change this at all from the exercises section? I'm wondering if we can break it out into a separate .js file and import it in both places so it's easier to manage.